docs/guides/cache.md

```diff
@@ -16,7 +16,16 @@ For a primer into Dockerfile caching see [this article](https://pythonspeed.com/
 
 Earthly cache is persisted in a docker volume called `earthly-cache` on your system. When Earthly starts for the first time, it brings up a BuildKit daemon in a Docker container, which initializes the `earthly-cache` volume. The volume is managed by Earthly's BuildKit daemon and there is a regular garbage-collection for old cache.
 
-## Specifying cache size
+{% hint style='info' %}
```

Comment:
I haven't seen this syntax in Markdown before. Have a link with more info? It doesn't seem to render in the GH (at least in the preview): https://github.com/earthly/earthly/pull/2065/files#diff-dac542612b20639dcadcd9b96e8d06d6c9da8eccb95b9403a4a1951a8515d734

Reply:
it's gitbook-specific; there's plenty of examples of it being used in [docs/earthfile/earthfile.md](https://raw.githubusercontent.com/earthly/earthly/main/docs/earthfile/earthfile.md)

Reply:
Ah ok. I didn't know we used Gitbook. Thanks.